auction-server/src/auction/service/auction_manager.rs

```diff
@@ -211,24 +221,40 @@ impl AuctionManager<Evm> for Service<Evm> {
             .send()
             .await?
             .tx_hash();
+        tracing::Span::current().record("tx_hash", tx_hash.to_string());
         Ok(tx_hash)
     }
 
+    #[tracing::instrument(skip_all, fields(bid_ids, tx_hash, auction_id, result))]
     async fn get_bid_results(
         &self,
         bids: Vec<entities::Bid<Evm>>,
         bid_status_auction: entities::BidStatusAuction<entities::BidStatusEvm>,
     ) -> Result<Option<Vec<entities::BidStatusEvm>>> {
+        tracing::Span::current().record(
+            "bid_ids",
+            format!(
+                "{:?}",
+                bids.iter()
+                    .map(|bid| bid.id.to_string())
+                    .collect::<Vec<String>>()
+            ),
+        );
+        tracing::Span::current().record("tx_hash", bid_status_auction.tx_hash.to_string());
+        tracing::Span::current().record("auction_id", bid_status_auction.id.to_string());
+
         let receipt = self
             .config
             .chain_config
             .provider
             .get_transaction_receipt(bid_status_auction.tx_hash)
             .await
             .map_err(|e| anyhow::anyhow!("Failed to get transaction receipt: {:?}", e))?;
+
         match receipt {
             Some(receipt) => {
                 let decoded_logs = Self::decode_logs_for_receipt(&receipt);
+                tracing::Span::current().record("result", format!("{:?}", decoded_logs));
```

Comment:
it seems that the default span attribute value length limit is around 2048 bytes. Maybe we should use a more compact representation here? Maybe just a true or false on tx failure?

Reply:
I tested it with a string with 1million chars. It worked :-?